### PR TITLE
Fixed calling varArg functions with one argument

### PR DESCRIPTION
**Pull request by dawidsula26**

### CODE REVIEW

File: interpreter/src/main/scala/pl/touk/nussknacker/engine/spel/internal/DefaultSpelConversionsProvider.scala [hunk 1]
```diff
@@ -23,10 +23,8 @@ class DefaultSpelConversionsProvider extends SpelConversionsProvider {
 
   override def getConversionService: GenericConversionService = {
     val service = new GenericConversionService
-    // We only need ObjectToArrayConverter, but it has package
-    // visibility so we add all collection converters.
-    DefaultConversionService.addCollectionConverters(service)
     service.addConverterFactory(new NumberToNumberConverterFactory())
+    service.addConverter(new ObjectToArrayConverter(service))
```
Comment (arkadius) [inline interpreter/src/main/scala/pl/touk/nussknacker/engine/spel/internal/DefaultSpelConversionsProvider.scala:27]: Please add notice in comment that it is for varargs purpose

Comment (dawidsula26) [reply interpreter/src/main/scala/pl/touk/nussknacker/engine/spel/internal/DefaultSpelConversionsProvider.scala:27]: What exactly should I test? Converting from object to array is something that is neccessary for other things to work, not a desired behaviour of ConversionProvider.

Comment (arkadius) [reply interpreter/src/main/scala/pl/touk/nussknacker/engine/spel/internal/DefaultSpelConversionsProvider.scala]: Test that you already have :) spel expression with single parameter in vararg - for example usage of String.format

**Review by arkadius — APPROVED** [review]

File: interpreter/src/main/scala/pl/touk/nussknacker/engine/spel/internal/DefaultSpelConversionsProvider.scala (hunk 1 above)
Comment (arkadius) [reply interpreter/src/main/scala/pl/touk/nussknacker/engine/spel/internal/DefaultSpelConversionsProvider.scala]: Intresting. I missed spel epxression tests. I see that it is included in this PR. Sorry for hasle

**Pull request merged**